app/assets/javascripts/govuk_publishing_components/components/step-by-step-nav.js

```diff
@@ -80,9 +80,22 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
       if (!thisel.querySelectorAll('.js-toggle-link').length) {
         var span = document.createElement('span')
-        span.className = 'gem-c-step-nav__toggle-link js-toggle-link'
-        span.setAttribute('aria-hidden', true)
-        span.setAttribute('hidden', 'hidden')
+        var showHideSpan = document.createElement('span')
+        var commaSpan = document.createElement('span')
+        var thisSectionSpan = document.createElement('span')
+
+        span.classList.add('gem-c-step-nav__toggle-link')
+        showHideSpan.classList.add('js-toggle-link')
+        commaSpan.classList.add('govuk-visually-hidden')
+        thisSectionSpan.classList.add('govuk-visually-hidden')
```

Comment:
For this line (and the three before it), you could just do `thisSectionSpan.className = 'govuk-visually-hidden'`